### ISSUE TITLE
Issues encountered during training

### ISSUE THREAD

**Issue by MingtaoGuo** [ali-vilab/VACE#83]
Hi VACE team,

I'm trying to reproduce the training script; however, I've encountered some issues.

I'm using the LTX-Video base model to train on face generation (512x512 resolution) with these modifications:

- Using only face keypoints for context
- DiT input is pure noise (reference image replaced with noise)
- Context-Adapter uses reference for first frame, then face keypoints for rest frames

After one day of training, I noticed:
1. Face colors look weird/wrong
2. Poses sometimes inaccurate 

Maybe because:
- LTX-Video's VAE compression is too aggressive?
- Or something wrong with my setup?

I'm using the same training loss from LTX-Video code and followed the VACE pipeline's LTX-Video inference part (except the changes mentioned above).

Did you guys encounter similar issues before? Any suggestions to fix this? Would adjusting the VAE help?

Thanks!

Left-to-right: Reference, Pose, Ground Truth, Results

https://github.com/user-attachments/assets/0d60eeaf-8ebe-4386-82df-68d2c4d91901

https://github.com/user-attachments/assets/7a35ede9-6336-4898-89f5-85c6597dd388

https://github.com/user-attachments/assets/d9a4ce55-e13f-4210-812d-73db6f04467b

**Comment by MingtaoGuo** [comment]
> These results look not too bad. If the training set scale is big enough, maybe you should continue training and see. LTX version VACE is experimental. It’s super fast, and it's not like you can demand much more in terms of its quality, right?

Thank you for your response. In your experiments, have you compared the final output quality between LTX-Video and Wan 1.3B models? My GPU has limited performance, so I can only use smaller models for testing.
